Libraries/LibKeyboard/CharacterMap.h

```diff
@@ -43,9 +43,13 @@ class CharacterMap {
 
     u32 get_char(KeyEvent);
     void set_character_map_data(CharacterMapData character_map_data);
+    void set_character_map_name(String& character_map_name);
```

Comment:
Please use a `const String&` here, as `set_character_map_name` does not intend to mutate the passed-in `String`.
Also, you can omit the parameter name sine it's already part of the function name :)